### ISSUE TITLE
Logging - warnings for ".registry.bad." files are emitted with "-r" switch

### ISSUE THREAD

**Issue by mwallner** [chocolatey/choco#1580]
## warnings for corrupted registry files are displayed with the `-r` switch

on each choco.exe call following message is getting emmited (twice)
```
A corrupt .registry file exists at C:\ProgramData\chocolatey\.chocolatey\accurev-client.5.7.1.170613\.registry.bad.
 Open this file in a text editor, and remove/escape any characters that
 are regarded as illegal within XML strings not surrounded by CData.
 These are typically the characters &, `<`, and `>`. Again, this
 is an XML document, so you will see many < and > characters, so just
 focus exclusively in the string values not surrounded by CData. Once
 these have been corrected, rename the .registry.bad file to .registry.
 Once saved, try running the same Chocolatey command that was just
 executed, so verify problem is fixed.
 NOTE: It will not be possible to rename the file in Windows Explorer.
 Instead, you can use the following PowerShell command:
 Move-Item .\.registry.bad .\.registry
```
the choco.exe parameter `-r` should not print warnings:
```
 -r, --limitoutput, --limit-output
     LimitOutput - Limit the output to essential information
```

**Comment by ferventcoder** [comment]
This was merged into stable at https://github.com/chocolatey/choco/commit/270641376c224777c1f17ee72bf31e5b8969974e

**Comment by ferventcoder** [comment]
Merged into stable to be released with 0.10.12.
